css/main.css

```diff
@@ -1,3 +1,87 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+
+.figuretop{
+    background-color: rgb(173, 126, 218);
+    padding: 25px;
+    margin: 0;
+} 
+
+.navbar{
+justify-content: space-around;
+background-color: blueviolet;
+}
+
+.list-elements{
+list-style-type: none;
```

Comment:
These can be formatted automatically by turning on this setting:
https://stackoverflow.com/questions/39494277/how-do-you-format-code-on-save-in-vs-code

Reply:
Hi Chris, forgot to populate here that I did automatic format for the script.